internal/controller/controllers/local_cluster_import_controller.go

```diff
@@ -0,0 +1,528 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	configv1 "github.com/openshift/api/config/v1"
+	hiveext "github.com/openshift/assisted-service/api/hiveextension/v1beta1"
+	aiv1beta1 "github.com/openshift/assisted-service/api/v1beta1"
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+	hivev1 "github.com/openshift/hive/apis/hive/v1"
+	"github.com/openshift/hive/apis/hive/v1/agent"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	clusterv1 "open-cluster-management.io/api/cluster/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	agentServiceConfigLocalClusterImportFinalizerName = "agentserviceconfig." + aiv1beta1.Group + "/local-cluster-import-deprovision"
+	localClusterImageSetName                          = "local-cluster-image-set"
+	hostKubeConfigsNamespace                          = "openshift-kube-apiserver"
+	hostKubeConfigsName                               = "node-kubeconfigs"
+	hostPullSecretNamespace                           = "openshift-config"
+	hostPullSecretName                                = "pull-secret"
```

Comment:
What about this is related to a "host"?

I think just remove the host prefix from all of these.